src/components/2019/year-in-review/Sports.js

```diff
@@ -0,0 +1,267 @@
+import React from 'react'
+import s from 'styled-components'
+
+import { MoreStories, Date, SectionMainText, PrimaryStory, StyledStoryLink, SecondaryStory, PrimaryStoryLine, SubSectionHeader, MainHeader } from './Typograph'
+
+const Wrapper = s.div`
+    /* Util classes */
+    .right-border {
+        border-right: 1px solid rgba(255, 255, 255, 0.3);
+    }
+
+    .right-border-inverse {
+        border-right: 1px solid rgba(0, 0, 0, 0.3);
+    }
+
+    .narrow-col-right {
+        padding-right: 24px;
+    }
+
+    .narrow-col-left {
+        padding-left: 24px;
+    }
+`
+
+const Section = s.div`
+    background-color: ${({ inverse }) => inverse ? "white" : "#131313"};
+    padding: ${({ inverse }) => inverse ? "80px 0px" : "120px 0px"};
+
+`
```

Comment:
would prefer extracting shared components like this between the section files into a common.js components file

Reply:
nvm, please move them into Typography